packages/im-ott/src/App.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 // Components
 import { ErrorBoundary } from "@sentry/react";
 import AppLoader from "imcomponents/molecules/appLoader";
+import { ToastContainer } from "imcomponents/atoms/toaster";
 // import SocketIoTest from "./SocketIoTest";
```

Comment:
remove commented code

Reply:
This was added intentionally to test socket.io, lets keep it there for a while until we decide not to go with socket and can be removed later